### ISSUE TITLE
issue with the z-index of modals

### ISSUE THREAD

**Issue by firstProjectHere** [Dogfalo/materialize#6497]
I was working on angular v4.0.0 with angular2-materialize and materialize-css(v0.98.2). When I upgraded angular application to version 6, I faced issues with the z-index of modals in the application.
I upgraded materialize-css version to 0.100.2 but it didn't work for me and then upgraded to v1.0.0 but no luck. Also, I tried to change the z-index with inline css but that inline css was not applied.
**Current scenario:** The second modal opens behind the first modal because z-index of second modal is less than first modal.
**Expected scenario:** The second modal should appear on top of first modal.

**Comment by pwcreative** [comment]
Why don't you just change the z-index of the second modal? This is not really an issue and should be closed.

**Comment by firstProjectHere** [comment]
Hi @pwcreative, I tried changing the z-index of first modal and then of second modal but things didn't work. Actually, the modal is not accepting the inline css.

**Comment by pwcreative** [comment]
Do you have a live link to the project?
Do you know what changed between angular 4 and 6 that may have caused the issue?
Do you absolutely have to use nested modals?

**Comment by firstProjectHere** [comment]
Yes, we are using modal based design UI. Following are the dependencies that are used
**Angular v4**
"dependencies": {
    "@angular/animations": "~4.3.1",
    "@angular/common": "~4.3.1",
    "@angular/compiler": "~4.3.1",
    "@angular/core": "~4.3.1",
    "@angular/forms": "~4.3.1",
    "@angular/http": "~4.3.1",
    "@angular/platform-browser": "~4.3.1",
    "@angular/platform-browser-dynamic": "~4.3.1",
    "@angular/platform-server": "~4.3.1",
    "@angular/router": "~4.3.1",
    "@angularclass/hmr": "~2.1.3",
    "@angularclass/hmr-loader": "^3.0.4",
    "angular2-grid": "^2.0.7",
    "angular2-materialize": "^6.7.2",
    "angular2-moment": "^1.7.0",
    "core-js": "^2.4.1",
    "hammerjs": "^2.0.8",
    "http-server": "^0.9.0",
    "ie-shim": "^0.1.0",
    "jquery": "^2.2.4",
    "materialize-css": "^0.98.2",
    "reflect-metadata": "^0.1.10",
    "rxjs": "^5.4.2",
    "rxjs-compat": "^6.4.0",
    "zone.js": "0.8.14"
  }

**After upgrading angular to v6,**
"dependencies": { 
    "@angular/animations": "^6.1.10", 
    "@angular/common": "^6.1.10", 
    "@angular/compiler": "^6.1.10", 
    "@angular/core": "^6.1.10", 
    "@angular/forms": "^6.1.10", 
    "@angular/http": "^6.1.10", 
    "@angular/platform-browser": "^6.1.10", 
    "@angular/platform-browser-dynamic": "^6.1.10", 
    "@angular/platform-server": "^6.1.10", 
    "@angular/router": "^6.1.10", 
    "@angularclass/hmr": "~2.1.3", 
    "@angularclass/hmr-loader": "^3.0.4", 
    "angular2-grid": "^2.0.7", 
    "angular2-materialize": "^15.1.10", 
    "angular2-moment": "^1.7.0", 
    "classlist.js": "^1.1.20150312", 
    "core-js": "^2.6.11", 
    "hammerjs": "^2.0.8", 
    "http-server": "^0.12.0", 
    "ie-shim": "^0.1.0", 
    "jquery": "^2.2.4", 
    "materialize-css": "^0.100.2", 
    "reflect-metadata": "^0.1.10", 
    "rxjs": "^6.5.4", 
    "rxjs-compat": "^6.5.4", 
    "zone.js": "^0.8.29" 
  }

**Comment by pwcreative** [comment]
I'd need to see a live example or codepen. If you can replicate the issue in a codepen then you may be able to take the solution through to the angular app, but it's impossible to help on current information given.
